jobs/sendEmailAlertNoFill.php

```diff
@@ -86,36 +85,32 @@ function sendEmail($email, $lastTaskDate, $from, $cc, $subject, $template) {
                 // User is not currently hired, skip it.
                 continue;
             }
-            $groups = $user->getGroups();
-            foreach ($groups as $group) {
-                if ($group->getName() == $ALL_USERS_GROUP) {
```

Comment:
Nice catch!

Reply:
I failed to catch it when I reviewed that code back in the day :sweat_smile: